_posts/2016-06-17-cynthia-intro.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Cynthia: introduction"
```

Comment:
Sorry - my bad, the requested format is: `Introducing DSSG Fellow $FirstName $LastName`